lib/node_modules/@stdlib/blas/base/cscal/benchmark/benchmark.js

```diff
@@ -65,12 +62,15 @@ function createBenchmark( len ) {
 	*/
 	function benchmark( b ) {
 		var viewX;
+		var cx;
 		var i;
 
 		b.tic();
 		for ( i = 0; i < b.iterations; i++ ) {
-			cscal( len, cx, x, 1 );
-			viewX = reinterpret( x, 0 );
+			cx = uniform( len*2, -100.0, 100.0, options );
+			cx = new Complex64Array( cx.buffer );
+			cscal( len, ca, cx, 1 );
+			viewX = reinterpret( cx, 0 );
```

Comment:
This is definitely not how you want to implement this benchmark. You are doing way too much in this loop, such that you won't actually be measuring the performance of `cscal`, but all the other things you are doing. This needs refactoring.